docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -2749,6 +2749,99 @@ Used as the default time zone for user preferences. Can be either `browser` for
 
 Set the default start of the week, valid values are: `saturday`, `sunday`, `monday` or `browser` to use the browser locale to define the first day of the week. Default is `browser`.
 
+### `[time_picker]`
+
+This section controls system-wide defaults for the time picker, such as the default quick ranges.
+
+#### `quick_ranges`
+
+Set the default set of quick relative offset time ranges that show up in the right column of the time picker. Each configuration entry should have a `from`, `to` and `display`field.
```

Comment:
Might be good just to do a one-off mention of it must be valid JSON? I don't think we have many JSON configuration values? 🤔